schema/project-level/codelists/votingRights.csv

```diff
@@ -0,0 +1,6 @@
+Code,Title,Description

```

Comment:
Why are the titles and descriptions missing from this codelist, whose codes match those in the OCDS shareholders extension?